internal/pkg/metadb/blob.go

```diff
@@ -138,7 +138,7 @@ func (b *BlobRef) Ready() error {
 // Returns an error if the current Status is not BlobRefStatusReady.
 func (b *BlobRef) MarkForDeletion() error {
 	if b.Status != BlobRefStatusInitializing && b.Status != BlobRefStatusReady {
-		return errors.New("Retire was called when Status is not either Initializing or Ready")
+		return errors.New("retire was called when Status is not either Initializing or Ready")
```

Comment:
This should be MarkForDeletion (I forgot to change it when I renamed the method name).
btw should a method name be written in small letters too?

Reply:
I'll leave it as MarkForDeletion for now as https://github.com/golang/go/wiki/CodeReviewComments#error-strings says the lower-case requirement doesn't apply if the error ends with punctuation.